src/disputeManager/DisputeManager.ts

```diff
@@ -267,15 +324,20 @@ class DisputeManager {
             disputeAuditingDataHash: disputeAuditingDataHash,
             disputer: disputer,
             timeout: timeoutStruct,
-            selfRemoval: selfRemoval
+            selfRemoval: selfRemoval,
+            latestInboundMessageBlockHash:
+                this.storage.inboundMessages.getLatestBlockHash() ||
+                ethers.ZeroHash,
+            lastInboundMessageBlockHeight:
+                this.storage.inboundMessages.getLatestBlockHeight() || 0
         };
 
         const outputSnapshotData =
             await this.diamondStateMachine.localDiamondContract.computeDisputeOutputSnapshotData.staticCall(
                 disputeInput,
-                latestStateSnapshot.toStruct(),
-                latestStateMachineState,
-                auditingData.genesisStateSnapshotData.latestJoinChannelBlockHash
+                auditingData.latestStateSnapshot,
+                auditingData.latestStateStateMachineState,
+                auditingData.inboundMessageBlocks || []
```

Comment:
uneeded `|| []`

getMessageBlocksInRange will return [] when the iterator is empty